### PR TITLE
Moar descriptions for things

### LINKED ISSUE #5664: Too many things lack descriptions!
#### Brief description of the issue
Many many atoms within the game lacks proper description. Just example, the biogenerator, microwaves, cargo consoles and many others lack one.
#### What you expected to happen
Some witty comment about this device mixed with some actually useful information. Or at least basic information.
#### Steps to reproduce
Inspect everything like crazy.

### PR DESCRIPTION
**Pull request by Shadow-Quill**
If you've got a suggestion, feel free to give it.
Closes #5664.

### CODE REVIEW

File: code/game/machinery/computer/supply.dm [hunk 1]
```diff
@@ -5,6 +5,7 @@
 // Supply requests console
 /obj/machinery/computer/supplycomp
 	name = "supply ordering console"
+	desc = "Request crates from here! Delivery not guarenteed."
```
Comment (Cerebulon) [inline code/game/machinery/computer/supply.dm]: *guaranteed (also in logbrowser.dm and rdconsole.dm)

File: code/modules/library/lib_machines.dm [hunk 2]
```diff
@@ -520,6 +521,7 @@ datum/borrowbook // Datum used to keep track of who has borrowed what when and f
  */
 /obj/machinery/bookbinder
 	name = "Book Binder"
+	desc = "Ties up a sheaf or paper into a nice book format."
```
Comment (Cerebulon) [inline code/modules/library/lib_machines.dm]: "Sheaf or paper"? I'm not sure Sheaf is the right word here?

Comment (Shadow-Quill) [reply code/modules/library/lib_machines.dm]: Meant to be `sheaf of`

**Comment by Shadow-Quill** [comment]
Had to webedit, as I haven't had time otherwise

**Pull request merged**